apps/site/models/datatype.py

```diff
@@ -10,8 +9,8 @@ class DataType(Base):
     DECIMAL = 5
     RATING = 6
 
-    name = models.CharField(max_length=255)
-    sql = models.CharField(max_length=500)
+    name = models.CharField(max_length=255, editable=False)
+    sql = models.CharField(max_length=500, editable=False)
 
```

Comment:
When you make model changes that are also schema changes, the codebase also needs the corresponding Django migrations (which allow us to easily update the schema on our already-existing database, Django-style). To do this:

```shell
$ cd /localground/apps
$ python manage.py makemigrations
$ python manage.py migrate
$ git add site/migrations #don't forget to add the new migration file to the repo
```
Please add the migration file and push it to this PR.